lexicon/bpe.py

```diff
@@ -28,6 +31,9 @@ def __init__(
         unk_label: str = "UNK",
         vocab_blacklist: Optional[Union[List[str], Set[str]]] = None,
         keep_special_lemmas: bool = True,
+        skip_unk_lemmas: bool = False,
+        add_all_bpe_phonemes: bool = True,
+        additional_words: Optional[tk.Path] = None,
```

Comment:
We could do it in this job, but my feeling is that adding a bunch of words with empty pronunciation is a separate task and I would rather have a `CreateEmptyPronunciationLexiconJob` or `CreateOrthOnlyLexiconJob` and then use the `MergeLexiconJob` to combine those.